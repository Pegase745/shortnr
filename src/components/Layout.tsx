import * as React from 'react';
import { Container } from 'semantic-ui-react';

interface ILayoutProps {
  children: React.ReactNode;
}

const Layout: React.FC<ILayoutProps> = ({ children }) => (
  <Container>{children}</Container>
);

export default Layout;
