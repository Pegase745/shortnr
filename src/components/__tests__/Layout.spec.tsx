import { shallow } from 'enzyme';
import * as React from 'react';

import Layout from '../Layout';

describe('Layout.tsx', () => {
  it('should render', () => {
    const app = shallow(<Layout>Test</Layout>);

    expect(app).toMatchSnapshot();
  });
});
