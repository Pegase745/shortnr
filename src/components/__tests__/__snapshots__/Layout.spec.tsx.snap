// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout.tsx should render 1`] = `
<Container>
  Test
</Container>
`;
